WalletWasabi.Fluent/Views/Wallets/Home/History/Columns/IndicatorsColumnView.axaml

```diff
@@ -47,11 +42,22 @@
         <PathIcon Opacity="0.6"
                   Classes.IncomingTransaction="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.IncomingTransaction}, Converter={x:Static converters:EnumToBoolConverter.Instance}}"
                   Classes.OutgoingTransaction="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.OutgoingTransaction}, Converter={x:Static converters:EnumToBoolConverter.Instance}}"
-                  Classes.SelfTransfer="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.SelfTransferTransaction}, Converter={x:Static converters:EnumToBoolConverter.Instance}}"
                   Classes.Coinjoin="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.Coinjoin}, Converter={x:Static converters:EnumToBoolConverter.Instance}}"
                   Classes.Cancellation="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.Cancellation}, Converter={x:Static converters:EnumToBoolConverter.Instance}}"
                   Classes.CoinjoinGroup="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.CoinjoinGroup}, Converter={x:Static converters:EnumToBoolConverter.Instance}}"
                   Classes.CPFP="{Binding Transaction.Type, ConverterParameter={x:Static model:TransactionType.CPFP}, Converter={x:Static converters:EnumToBoolConverter.Instance}}">
+          <Classes.SelfTransfer>
+            <MultiBinding Converter="{x:Static BoolConverters.And}">
+              <Binding Path="Transaction.Type" ConverterParameter="{x:Static model:TransactionType.SelfTransferTransaction}" Converter="{x:Static converters:EnumToBoolConverter.Instance}" />
+              <Binding Path="!HasBeenSpedUp" />
+            </MultiBinding>
+          </Classes.SelfTransfer>
+          <Classes.SpedUpSelfTransfer>
+            <MultiBinding Converter="{x:Static BoolConverters.And}">
+              <Binding Path="Transaction.Type" ConverterParameter="{x:Static model:TransactionType.SelfTransferTransaction}" Converter="{x:Static converters:EnumToBoolConverter.Instance}" />
+              <Binding Path="HasBeenSpedUp" />
+            </MultiBinding>
+          </Classes.SpedUpSelfTransfer>
```

Comment:
@SuperJMN, in order to fix the issue mentioned here https://github.com/WalletWasabi/WalletWasabi/pull/13082#issuecomment-2156097440, you just need to remove the `SpedUpSelfTransfer` style class altogether. And remove the `<Binding Path="!HasBeenSpedUp" />` condition from `SelfTransfer` style class.

Reply:
Please, review if it's working properly now. Thanks for the suggestions, BTW!